src/Fluid/Flu_FixUp_Restrict.cpp

```diff
@@ -297,7 +297,7 @@ void Flu_FixUp_Restrict( const int FaLv, const int SonFluSg, const int FaFluSg,
 
 //    check the minimum pressure/internal energy and, when the dual-energy formalism is adopted, ensure the consistency between
 //    pressure, total energy density, and the dual-energy variable
-#     if ( MODEL == HYDRO )
+#     if ( MODEL == HYDRO && !defined SRHD )
```

Comment:
Please also update the comment at line 347 to be `# endif // #if ( MODEL == HYDRO  &&  !defined SRHD )`, and there should be two spaces around `&&` to match the code style.